test/controllers/api/v2/auth_source_internals_controller_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class Api::V2::AuthSourceInternalsControllerTest < ActionController::TestCase
+  test "should get index" do
+    get :index, { }
+    assert_response :success
+    assert_not_nil assigns(:auth_source_internals)
+    auth_source_internals = ActiveSupport::JSON.decode(@response.body)
+    refute auth_source_internals.empty?
+  end
+
+  test "should show auth_source_internal" do
+    get :show, { :id => auth_sources(:internal).to_param }
+    assert_response :success
+    show_response = ActiveSupport::JSON.decode(@response.body)
+    assert !show_response.empty?
```

Comment:
it would be good to assert we got the right record, e.g. by
```
assert_equal auth_sources(:internal).name, show_response['name']
```
or by using id